src/ray/raylet/scheduling/local_resource_manager.cc

```diff
@@ -358,8 +358,6 @@ bool LocalResourceManager::AllocateTaskResourceInstances(
       return false;
     }
   }
-
-  OnResourceChanged();
```

Comment:
Is it okay to remove this?

Reply:
ah this is required to notify changes to the subscriber

Reply:
Is this safe to remove?

Reply:
We need to notify the change for all public APIs, so this can't be removed.